glass/lensing.py

```diff
@@ -61,27 +61,29 @@ def from_convergence(  # noqa: PLR0913
     deflection potential, deflection, and shear maps. The maps are
     computed via spherical harmonic transforms.
 
-    Returns the maps of:
-
-    * deflection potential if ``potential`` is true.
-    * potential (complex) if ``deflection`` is true.
-    * shear (complex) if ``shear`` is true.
-
     Parameters
     ----------
-    kappa:
+    kappa
         HEALPix map of the convergence field.
-    lmax:
+    lmax
         Maximum angular mode number to use in the transform.
-    potential:
+    potential
         Which lensing maps to return.
-    deflection:
+    deflection
         Which lensing maps to return.
-    shear:
+    shear
         Which lensing maps to return.
-    discretized:
+    discretized
         Correct the pixel window function in output maps.
 
+    Returns
+    -------
+        The maps of:
+
+        * deflection potential if ``potential`` is true.
+        * potential (complex) if ``deflection`` is true.
+        * shear (complex) if ``shear`` is true.
+
     Notes
```

Comment:
Nitpicking, but "Notes" and "References" in the rendered docs do not get a `:` by default -

<img width="348" alt="image" src="https://github.com/user-attachments/assets/f2797b48-55b2-4072-b4c6-1bee7ed29c63">

Should we add it in the docstring, or just leave it as it is?

Reply:
I had spotted this. I'd be inclined to follow the NumpyDoc standard https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard